flake.nix

```diff
@@ -14,16 +14,34 @@
 
   outputs = { self, flake-compat, nixpkgs, utils }: utils.lib.eachDefaultSystem (system: let
     pkgs = import nixpkgs { inherit system; };
-  in {
+  in rec {
+
+    packages.ast-checks = pkgs.rustPlatform.buildRustPackage {
+      name = "ast-checks";
+      cargoSha256 = "130qcmra644xmixmyrskwbx40r9fbbzql3gc35y3vq4bz9jgg81n";
+      src = ./ast-checks;
+    };
+
+    # Find all of the binaries installed by ast-checks. Note, if this changes
+    # in the future to use wrappers or something else that pollute the bin/
+    # directory, this logic will have to grow
+    ast-check-names = let
```

Comment:
This should probably by in the let binding above, otherwise `nix flake check` will fail.